products/esa-griml-iml-fv1/collection.json

```
@@ -0,0 +1,120 @@
+    "links": [
+      {
+        "rel": "root",
+        "href": "../../catalog.json",
+        "type": "application/json",
+        "title": "Open Science Catalog"
+      },
+      {
+        "rel": "via",
+        "href": "https://doi.org/10.22008/FK2/MBKW9N",
+        "title": "Access"
+      },
+      {
+        "rel": "via",
+        "href": "https://doi.org/10.1038/s41598-021-83509-1",
+        "title": "Documentation"
+      },
+      {
+        "rel": "parent",
+        "href": "../catalog.json",
+        "type": "application/json",
+        "title": "Products"
+      },
+      {
+        "rel": "related",
+        "href": "../../projects/examining-greenland-s-ice-marginal-lakes-under-a-changing-climate-griml/collection.json",
+        "type": "application/json",
+        "title": "Project: (LPF) GRIML"
+      },
+      {
+        "rel": "related",
+        "href": "../../themes/cryosphere/catalog.json",
+        "type": "application/json",
+        "title": "Theme: Cryosphere"
+      },
+      {
+        "rel": "related",
+        "href": "../../variables/lake-surface-temperature/catalog.json",
+        "type": "application/json",
+        "title": "Variable: Lake Surface Temperature"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/landsat-8/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Landsat-8"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/landsat-9/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Landsat-9"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/sentinel-1/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Sentinel-1"
+      },
+      {

```

Suggestion: add Sentinel-2 reference here also, as Sentinel-2 is another key dataset used.

```suggestion
     {
        "rel": "related",
        "href": "../../eo-missions/sentinel-2/catalog.json",
        "type": "application/json",
        "title": "EO Mission: Sentinel-2"
      },

```